### PR TITLE
Fix ShakeOnDeath and remove hardcoded screen shaking from bridges

### PR DESCRIPTION
**Pull request by abcdefg30**
`ShakeOnDeath` was using the `Intensity` parameter for the duration all along. Fixed that and introduced a new `Duration` parameter together with an update rule for this change.

To make use of the new fix, I also removed the screen shaking from `Bridge` and `GroundLevelBridge` and made bridge actors use `ShakeOnDeath` instead. Note that the update rule for this change has a dependency on #15107 or - if this gets merged first - #15107 will need to adjust the rule.

### CODE REVIEW

**Comment by reaperrr** [comment]
👍 from me. #15107 was merged, so you can solve the TODO now.

**Review by GraionDilach — APPROVED** [review]
:+1: so LGTM after the rebase.

**Comment by abcdefg30** [comment]
Rebased & updated.

File: OpenRA.Mods.Common/UpdateRules/Rules/ChangeIntensiyToDuration.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class ChangeIntensiyToDuration : UpdateRule
```
Comment (reaperrr) [inline OpenRA.Mods.Common/UpdateRules/Rules/ChangeIntensiyToDuration.cs]: Intensi**t**y (probably copy-paste error)

File: OpenRA.Mods.Common/OpenRA.Mods.Common.csproj [hunk 2]
```diff
@@ -578,6 +578,8 @@
     <Compile Include="Traits\World\WeatherOverlay.cs" />
     <Compile Include="Traits\World\ActorSpawnManager.cs" />
     <Compile Include="Traits\ActorSpawner.cs" />
+    <Compile Include="UpdateRules\Rules\AddShakeToBridge.cs" />
+    <Compile Include="UpdateRules\Rules\ChangeIntensiyToDuration.cs" />
```
Comment (reaperrr) [inline OpenRA.Mods.Common/OpenRA.Mods.Common.csproj]: Intensi**t**y (probably copy-paste error)

**Review by reaperrr — DISMISSED** [review]

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
The intensity rule was also missing from UpdatePath, but I was bored and decided to fix these myself so we can merge.

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/027927eda7520298a78730619edc417f6202b995...a854b7e2e484d1f52045270dea1b9d8d22acf805

**Pull request merged**